src/utils/url.ts

```diff
@@ -28,8 +29,12 @@ export function isURLInList(url: string, list: string[]) {
  * @param urlTemplate URL template ("google.*", "youtube.com" etc).
  */
 export function isURLMatched(url: string, urlTemplate: string): boolean {
-    const regex = createUrlRegex(urlTemplate);
-    return Boolean(url.match(regex));
+    if (url.indexOf('[') === -1 || urlTemplate.indexOf('[') === -1) {
```

Comment:
Better use `String.includes`. I think that IP address is a rare case. IPv6 is even more rear case. Let's simplify it. Let's fast check if both URLs are IPv6 addresses. If some of them is, and other not, we can return `false`.